src/components/scrim/edit-scrim-form.tsx

```diff
@@ -205,11 +217,55 @@ export function EditScrimForm({
                 <FormDescription>
                   This is the team that will be associated with this scrim. You
                   can assign scrims to teams that you manage or own.
-                </FormDescription>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
+              </FormDescription>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        
+        <FormField
+          control={form.control}
+          name="date"
+          render={({ field }) => (
+            <FormItem className="flex flex-col">
+              <FormLabel>Scrim Date</FormLabel>
+              <Popover>
+                <PopoverTrigger asChild>
+                  <FormControl>
+                    <Button
+                      variant="outline"
+                      className={cn(
+                        "w-[240px] pl-3 text-left font-normal",
+                        !field.value && "text-muted-foreground"
+                      )}
+                    >
+                      {field.value ? (
+                        format(new Date(field.value), "PPP")
+                      ) : (
+                        <span>Edit date</span>
+                      )}
+                      <CalendarIcon className="ml-auto h-4 w-4 opacity-50" />
+                    </Button>
+                  </FormControl>
+                </PopoverTrigger>
+                <PopoverContent className="w-auto p-0" align="start">
+                  <Calendar
+                    mode="single"
+                    selected={field.value}
+                    onSelect={field.onChange}
+                    disabled={(date) =>
+                      date > new Date() || date < new Date("2016-01-01")
+                    }
+                  />
+                </PopoverContent>
+              </Popover>
+              <FormDescription>
+                The scrim date is the date when the scrim took place.
```

Comment:
Replace this text with `The date when the scrim took place.` as the current text is redundant. See the other fields for writing style.